docs/sources/datasources/alertmanager/_index.md

```diff
@@ -28,24 +28,27 @@ When using Prometheus, contact points and notification policies are read-only in
-| **Alertmanager**           |                                                                                                                                                                                                               |
-| **Implementation**         | Alertmanager implementation. **Mimir**, **Cortex,** and **Prometheus** are supported                                                                                                                          |
-| **Receive Grafana Alerts** | When enabled the Alertmanager receives alert instances from Grafana-managed alert rules. **Important:** It works only if Grafana alerting is configured to send its alert instances to external Alertmanagers |
-| **HTTP**                   |                                                                                                                                                                                                               |
-| **URL**                    | Sets the HTTP protocol, IP, and port of your Alertmanager instance, such as `https://alertmanager.example.org:9093`                                                                                           |
-| **Access**                 | Only **Server** access mode is functional                                                                                                                                                                     |
-|                            |                                                                                                                                                                                                               |
+1. Set the data source's basic configuration options:
+
+   | Name                       | Description                                                                                                                                                                                                   |
+   | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | **Name**                   | Sets the name you use to refer to the data source                                                                                                                                                             |
+   | **Default**                | Sets whether the data source is pre-selected for new panels and queries                                                                                                                                       |
+   | **Alertmanager**           |                                                                                                                                                                                                               |
```

Comment:
Unrelated to your changes but I think that this table has rows that need to be combined.

I think the rows with the name **Alertmanager** and **Implementation** should be combined, and also the rows **HTTP** and **URL**

Reply:
Fixed!